### PR TITLE
fix: fix non-binary files to be unchecked in check_caret_rclcpp

### PR DESCRIPTION
**Pull request by atsushi421**
This PR modifies non-binary files to be unchecked in `ros2 caret check_caret_rclcpp` command.
This bug was reported in the following ticket: 
https://tier4.atlassian.net/browse/T4PB-22214

### CODE REVIEW

**Comment by atsushi421** [comment]
@hsgwa @takeshi-iwanari 
I have fixed the bug reported in [this ticket](https://tier4.atlassian.net/browse/T4PB-22214), can you please review this?
Also, I have fixed minor errors in mypy test.

**Review by takeshi-iwanari — APPROVED** [review]
LGTM!
Thank you for fixing it!!
I confirmed warning messages are disappeared

File: ros2caret/verb/check_caret_rclcpp.py [hunk 1]
```diff
@@ -130,20 +134,24 @@ def _get_file_paths(dir_path: str) -> List[str]:
...
-        return (subprocess.Popen(cmd,
-                                 stdout=subprocess.PIPE,
-                                 shell=True).communicate()[0]
-                )
+        stdout = (subprocess.Popen(cmd,
+                                   stdout=subprocess.PIPE,
+                                   shell=True).communicate()[0]
+                  )
+
+        return True if stdout else False
```
Comment (hsgwa) [inline ros2caret/verb/check_caret_rclcpp.py:142]: How about the following implementation?
```suggestion
        return bool(stdout)
```

File: ros2caret/verb/check_caret_rclcpp.py [hunk 2]
```diff
@@ -130,20 +134,24 @@ def _get_file_paths(dir_path: str) -> List[str]:
...
-                                 shell=True
-                                 ).communicate()[0]
-                ).decode('utf-8')
+        stdout = (subprocess.Popen(cmd,
+                                   stdout=subprocess.PIPE,
+                                   shell=True
+                                   ).communicate()[0]
+                  ).decode('utf-8')
+
+        return True if stdout else False
```
Comment (hsgwa) [inline ros2caret/verb/check_caret_rclcpp.py:154]: How about the following implementation?
```suggestion
        return bool(stdout)
```

File: ros2caret/verb/check_caret_rclcpp.py (hunk 1 above)
Comment (hsgwa) [reply ros2caret/verb/check_caret_rclcpp.py:142]: @atsushi421 
friendly ping

Comment (atsushi421) [reply ros2caret/verb/check_caret_rclcpp.py]: @hsgwa 
Thank you for pointing that out.
Fixed in: Thank you very much for pointing that out. :bow:

**Review by hsgwa — APPROVED** [review]
LGTM

**Pull request merged**